tests/test_docker.sh

```diff
@@ -10,10 +10,18 @@ assert_up_and_running() {
     rlRun -t -c "curl -k -L -o- https://$IP_ADDRESS:8443/ | grep 'Welcome to Kiwi TCMS'"
 }
 
+assert_redirect_to_initdb() {
+    sleep 10
+    IP_ADDRESS=`docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' kiwi_web`
+    # HTTPS displays the init-db page
+    rlRun -t -c "curl -k -L -o- https://$IP_ADDRESS:8443/ | grep 'Initialize database'"
+}
+
 rlJournalStart
     rlPhaseStartTest "[PostgreSQL] Container up"
         rlRun -t -c "docker-compose -f docker-compose.postgres up -d"
         sleep 10
+        assert_redirect_to_initdb
         rlRun -t -c "docker exec -i kiwi_web /Kiwi/manage.py migrate"
```

Comment:
This change partially validates the new functionality. It makes sure that the init-db page is shown but doesn't actually exercise it. You can try simulating a POST with curl and removing `manage.py migrate`.